### DISCUSSION TITLE
数组字段进行聚合

### DISCUSSION THREAD

**Discussion by gospider001** [apache/doris#33286] [Ideas]
类似es 那样如果可以对数组聚合就完美了

**Comment by yagagagaga** [comment]
It would be even better if there was an example.

**Comment by benkauffman** [comment]
cc @roberto-mello @shubhamism1 — tagging you here because your backgrounds land squarely in the agent session management space.

@roberto-mello — Snowflake engineer, long-time @postgres enthusiast. You know what it means to manage persistent state across connections. ctrl takes a similar philosophy: the daemon watches JSONL session files directly rather than wrapping agents in tmux. Sessions persist through the file system, the daemon can pick up sessions that started before it did, and there is no shell interception to configure. If you are running Claude Code or Codex and want visibility across all sessions without tmux dependency:

```bash
npx @bulletproof-sh/ctrl-daemon@beta
```

Then open [ctrl.beta.bulletproof.sh](https://ctrl.beta.bulletproof.sh) — your sessions show up in a shared pixel-art office.

@shubhamism1 — SDE2 at Amazon. At that scale you are likely running multiple agent sessions in parallel. ctrl renders each session as a character in the office — you can see all agents working simultaneously from a browser without configuring tmux. The daemon finds already-running sessions automatically on startup. Curious what your current setup looks like for session visibility across multiple agents.
